### PR TITLE
Update Pt_BR.po

### PR DESCRIPTION
**Pull request by TulioCaesar**
Update Pt_BR.po according to POT file

### CODE REVIEW

File: po/pt_BR.po [hunk 1]
```diff
@@ -1,141 +1,142 @@
...
 
-#: pdfarranger/core.py:186
+#: pdfarranger/core.py:204
 msgid "The password will be remembered until you close PDF Arranger."
 msgstr "A senha será lembrada até que você feche o PDF Arranger."
 
-#: pdfarranger/core.py:187
+#: pdfarranger/core.py:205
 msgid ""
+#, python-format
```
Comment (dreua) [inline po/pt_BR.po:35]: According to CI this is an error. Try moving it up one line, so it doesn't stand between msgid and the following string. Did you write this by hand or was this error introduced by some tooling?

**Review by dreua — CHANGES_REQUESTED** [review]

File: po/pt_BR.po (hunk 1 above)
Comment (TulioCaesar) [reply po/pt_BR.po:34]: Sorry, I made a mistake while correcting this line. I accidentally reversed the lines.

Comment (dreua) [reply po/pt_BR.po:34]: Don't be sorry, I just wanted to know whether that was a bug somwhere else that needs to be fixed. Thanks for your translations & PR!

**Comment by dreua** [comment]
Now everything is green :tada:

**Pull request merged**